services/federator/test/unit/Test/Federator/Remote.hs

```diff
@@ -132,6 +132,10 @@ testValidatesCertificateWrongHostname =
             Left (RemoteError _ (FederatorClientTLSException _)) -> pure ()
             Left x -> assertFailure $ "Expected TLS failure, got: " <> show x
             Right _ -> assertFailure "Expected connection with the server to fail",
+      -- @SF.Federation @TSFI.RESTfulAPI @S2
+      --
+      -- Refuse to connect with a server when it has no X509v3 Extended Key
+      -- Usage extension in the certificate.
```

Comment:
The END tag is missing here

Reply:
In that earlier ticket and PR, I did not have an END tag for a nested test case like this. If I try to add it here, because of automatic formatting we have in the `wire-server` codebase it gets weird indentation and it does not match indenting of that introductory @-tag: instead of 6 spaces, it gets 12 spaces. There is no way around it with this automatic formatting in place as far as I know.

Therefore, can you make it work without the @END tag? If not, I'll see what I can do with this automatic formatting.

Reply:
Unfortunately the parser does not work without the END tag. What would happen if you add the comment in front of the line `testValidatesCertificateWrongHostname`? Would it work to then with the automatic formatting?

Reply:
> Unfortunately the parser does not work without the END tag. What would happen if you add the comment in front of the line `testValidatesCertificateWrongHostname`? Would it work to then with the automatic formatting?

Sure, I've placed the tags around the top-level code. Let me know if this cuts it.

Reply:
Works like a charm! Thanks @mdimjasevic